main.go

```diff
@@ -81,6 +81,31 @@ func main() {
 		exit = true
 	}
 
+	if opts.Edit != nil {
+		editor := os.Getenv("EDITOR")
+		editors := []string{editor, "nano", "vim", "vi"}
+		for _, ed := range editors {
+			editor, err = exec.LookPath(ed)
+			if err == nil {
+				break
+			}
+		}
+		if editor != "" {
+			configDir, _ := os.UserConfigDir()
+			if *opts.Edit == "" {
+				*opts.Edit = "dxhd.sh"
+			}
+			path := filepath.Join(configDir, "dxhd", *opts.Edit)
```

Comment:
Add a new function in `config.go`, and make it return the default config Directory, and use it in [this](https://github.com/dakyskye/dxhd/blob/master/config/config.go#L9) function and here, instead of `os.UserConfigDir()`.